source/configuration.rst

```diff
@@ -250,15 +250,14 @@ The only required configuration parameter is an execution target, which depends
 .. _ssl-keystore:
 
 ``ssl-keystore`` (Available since GLPI Agent v1.11)
-    This option is only usable on Windows or MacOSX.
-
-    Keystore support on Windows and Keychain support on MacOSX are enabled by default
-    to provide a way to authentify SSL GLPI server if CA certificate or server certificate
-    is integrated there.
+    Keystore support on Windows, Keychain support on MacOSX and CA trust store support on Unix/Linux 
+    are enabled by default to provide a way to authentify SSL GLPI server if CA certificate or server 
+    certificate is integrated there. The default CA bundle from Mozilla's will be included only if the
+    GLPI-Agent is unable to extract certificates from the operating system. (this support case is available since GLPI Agent v1.12)
```

Comment:
As discussed in the PR, other OS are not concerned. You don't need to change this paragraph.

Reply:
It was partially approved and I reverted some not required changes as I said in my last comments. My comment here are following what I did to merge manually your glpi-agent PR.

Reply:
You still have to remove this change.